src/pages/home/sidebar/SidebarLinks.js

```diff
@@ -167,20 +191,10 @@ class SidebarLinks extends React.Component {
         }
 
         const activeReportID = parseInt(this.props.currentlyViewedReportID, 10);
-
-        const {recentReports} = getSidebarOptions(
-            this.props.reports,
-            this.props.personalDetails,
-            this.props.draftComments,
-            activeReportID,
-            this.props.priorityMode,
-            this.props.betas,
-        );
-
         const sections = [{
             title: '',
             indexOffset: 0,
-            data: recentReports,
+            data: this.state.orderedReports ?? [],
```

Comment:
Unfortunately, we also encourage against using `??` in our style guidelines. sorry for not catching it previously.
```suggestion
            data: this.state.orderedReports || [],
```